src/telemetry/telemetry.controller.ts

```diff
@@ -63,7 +67,13 @@ export class TelemetryController {
     if (graffiti) {
       const user = await this.usersService.findByGraffiti(graffiti);
       if (user) {
-        await this.nodeUptimes.upsert(user);
+        const uptime = await this.nodeUptimes.upsert(user);
+        if (uptime && uptime.total_hours >= NODE_UPTIME_CREDIT_HOURS) {
+          await this.graphileWorkerService.addJob(
+            GraphileWorkerPattern.CREATE_NODE_UPTIME_EVENT,
+            { userId: user.id },
+          );
+        }
```

Comment:
We might need to move this into the graphile job we added here in another PR, or else we may need a lock here?